app/tests/e2e/test_connections.py

```diff
@@ -39,6 +43,10 @@ async def test_accept_invitation(
     )
     connection_record = accept_response.json()
 
+    assert check_webhook_state(
+        client=alice_member_client, desired_state={"state": "active"}
+    )
```

Comment:
YES!! this makes the tests so much better.

Nit: this check should be placed below the next two asserts as it comes after request-sent

Reply:
Shouldn't we check which record id / topic we want to check the state for? Theoretically if there is a connection record with state=active this will succeed right?

Reply:
well spotted and good question. It is taken care of though (implicitly). the method extracts the wallet-id and gets the connection records only for that wallet. the wallet is crreated (and destroyed) per test run and stems from the fixture so there is only that connections between alice and bob. Could add checking against the connection ID though to be more explicit, maybe that's better/clearer